### PR TITLE
Service Dashboard Endpoint Sidepanel

### PR DESCRIPTION
**Pull request by MikeShi42**
image

### CODE REVIEW

**Review by jaggederest — COMMENTED** [review]
Looks great, I'll leave it for a more detailed review by someone else, but I'm learning a lot 👍

File: packages/api/src/clickhouse/index.ts [hunk 1]
```diff
@@ -1516,6 +1516,112 @@ export const getMultiSeriesChartLegacyFormat = async ({
...
+  const tableName = getLogStreamTableName(tableVersion, teamId);
+
+  const parentSpanWhereCondition = await buildSearchQueryWhereCondition({
+    endTime,
+    propertyTypeMappingsModel,
+    query: parentSpanWhere,
+    startTime,
+  });
+
+  const childrenSpanWhereCondition = await buildSearchQueryWhereCondition({
```
Comment (wrn14897) [inline packages/api/src/clickhouse/index.ts:1548]: nit: can do this concurrently ?

**Review by wrn14897 — APPROVED** [review]
🚢 🚢 🚢

**Comment by wrn14897** [comment]
Can you also add changeset ?

**Review by wrn14897 — APPROVED** [review]

**Pull request merged**